revolv/static/project.scss

```diff
@@ -77,3 +510,207 @@
     height: 300px;
     margin: 0 20px 20px 20px;
 }
+
+
+@media screen and (max-width: 1050px) {
```

Comment:
For these media queries, you can use foundation builtin stuff, like `@media #{$small-only} and (orientation: portrait) {` instead of `@media screen and (max-width: 1050px) {`. Check out home.scss for examples